games/templates/play_game_as_player.html

```diff
@@ -1,66 +1,122 @@
 {% extends "base.html" %}
 
 {% block content %}
-{% if turn %}
-<h1>{{ player }} (Your turn - {{ turn.get_state_display }})</h1>
-<form method="POST" action="{% url 'turns_end_phase' turn_id=turn.pk %}">
-    {% csrf_token %}
-    <button>End Phase</button>
-</form>
-<table class="table">
-    <tbody>
-      <tr>
-        <td>Actions</td>
-        <td>{{ turn.available_actions }}</td>
-      </tr>
-      <tr>
-        <td>Buys</td>
-        <td>{{ turn.available_buys }}</td>
-      </tr>
-      <tr>
-        <td>Money</td>
-        <td>{{ turn.available_money }}</td>
-      </tr>
-    </tbody>
-</table>
-{% else %}
-<h1>{{ player }} (Not your turn)</h1>
-{% endif %}
-<h2>HAND</h2>
-<table class="table">
-    <tbody>
-      {% for card in deck.real_hand %}
-      <tr>
-        <td>{{ card }}</td>
-        <td><button>Play</button></td>
-      </tr>
-      {% endfor %}
-    </tbody>
-</table>
-<ul>
+<nav class="navbar navbar-light bg-light">
+  <span class="navbar-brand mb-0 h1">
+      {{ player }}
+      {% if turn %}
+      (Your turn - {{ turn.get_state_display }})
+      {% else %}
+      (Not your turn)
+      {% endif %}
+  </span>
+</nav>
+<div class="flex">
+    <div id="right-sidebar" class="sidebar">
+        {% if turn %}
+        <div id="actions">Actions ({{ turn.available_actions }})</div>
+        <div id="buys">Buys ({{ turn.available_buys }})</div>
+        <div id="money">Money ({{ turn.available_money }})</div>
+        <div id="end-phase">
+            <form method="POST" action="{% url 'turns_end_phase' turn_id=turn.pk %}">
+                {% csrf_token %}
+                <button>End Phase</button>
+            </form>
+        </div>
+        {% endif %}
+    </div>
+    <div id="body">
+        <h2>KINGDOM</h2>
+        <span class="card">
+            {{ game.real_kingdom.Gold.card }}
+            </br>
+            &nbsp;
+        </span>
+        <span class="card">
+            {{ game.real_kingdom.Silver.card }}
+            </br>
+            &nbsp;
+        </span>
+        <span class="card">
+            {{ game.real_kingdom.Copper.card }}
+            </br>
+            &nbsp;
+        </span>
+        <span class="card">
+            {{ game.real_kingdom.Province.card }}
+            </br>
+            ({{ game.real_kingdom.Province.count }})
+        </span>
+        <span class="card">
+            {{ game.real_kingdom.Duchy.card }}
+            </br>
+            ({{ game.real_kingdom.Duchy.count }})
+        </span>
+        <span class="card">
+            {{ game.real_kingdom.Estate.card }}
+            </br>
+            ({{ game.real_kingdom.Estate.count }})
+        </span>
+        <span class="card">
+            {{ game.real_kingdom.Curse.card }}
+            </br>
+            ({{ game.real_kingdom.Curse.count }})
+        </span>
+        </br>
+        {% for row in game.real_kingdom.values %}
+            {% if row.card.name in "Gold,Silver,Copper,Province,Duchy,Estate,Curse" %}
```

Comment:
We probably ought to make helpers to auto-pull the different sets of kingdom cards (money is infinite, victory + curses are a thing, the rest of the cards). But I tried to mostly only do UI work in this PR. And that will require some amount of backend work.